

try:
    pint(222)
except Exception as e:
    print(e)